packages/components/pie-thumbnail/src/thumbnail.scss

```
@@ -16,6 +17,34 @@
     --thumbnail-border-color: var(--dt-color-border-default);
   }
 
+  &.c-thumbnail--disabled {
+    opacity: 0.5;
+  }
+
+  &.c-thumbnail--background-subtle {
+    --thumbnail-bg-color: var(--dt-color-container-subtle);
+  }
+
+  &.c-thumbnail--background-strong {
+    --thumbnail-bg-color: var(--dt-color-container-strong);
+  }
+
+  &.c-thumbnail--background-dark {
+    --thumbnail-bg-color: var(--dt-color-container-dark);
+  }
+
+  &.c-thumbnail--background-inverse {
+    --thumbnail-bg-color: var(--dt-color-container-inverse);
+  }
+
+  &.c-thumbnail--background-inverse-alternative {
```

As dashes are used to indicate child elements, can just use camelCase for the background classes:

```suggestion
  &.c-thumbnail--backgroundSubtle {
    --thumbnail-bg-color: var(--dt-color-container-subtle);
  }

  &.c-thumbnail--backgroundStrong {
    --thumbnail-bg-color: var(--dt-color-container-strong);
  }

  &.c-thumbnail--backgroundDark {
    --thumbnail-bg-color: var(--dt-color-container-dark);
  }

  &.c-thumbnail--backgroundInverse {
    --thumbnail-bg-color: var(--dt-color-container-inverse);
  }

  &.c-thumbnail--backgroundInverseAlt {
```

---

i think having them as `camelCase` will make it harder to construct the class name.
for example, the prop value is `inverse-alt` (all of our prop values are snake-case), to create the className `backgroundInverseAlt`, we need a transformer and I don't think we follow this approach in the code base.

---

how about ` &.c-thumbnail-background--subtle `

---

updated to use the last suggestion